test/spec/ol/format/esrijsonformat.test.js

```diff
@@ -304,15 +304,15 @@ describe('ol.format.EsriJSON', function() {
               -10579560.16462974, 4716567.373073828]))
             .to.be(true);
 
-        var last = result[305];
+        var last = result[8];
```

Comment:
I did, it's here: test/spec/ol/format/esrijson/ksfields.json just on top of this